pymc/distributions/continuous.py

```diff
@@ -1875,6 +1885,22 @@ def logcdf(value, nu, mu, sigma):
             msg="nu > 0, sigma > 0",
         )
 
+    def icdf(value, nu, mu, sigma):
+        _, sigma = get_tau_sigma(sigma=sigma)
```

Comment:
This is useless?

Reply:
It looks like so, I kept it because the logcdf function above also uses it in the same way. 
Because we are testing from tau as well in the test suite, I thought it would be part of the tau/sigma conversion, or that it initialize sigma = 1 whenever neither sigma and tau are not passed as arguments. 
However, when I take it out, the tests pass anyways, so I think it is useless.
Removed in the commit https://github.com/pymc-devs/pymc/pull/6845/commits/be24e627a698659037db8cb8affb9c82ad24d2ee